src/Pages/Reputation.js

```diff
@@ -86,34 +86,35 @@ export default class Reputation extends Component{
         var row_stop = this.state.current_page * this.state.rows_per_page;
 
         return (
-            <Card className="shadow p-2 mb-2 bg-white rounded" style={{height: "85vh"}}>
-                <Card.Body className="pt-3 pb-0">
-                    <Card.Text>
-                        <Table hover responsive>
-                            <thead>
-                                <tr style={{"line-height": "25px"}}>
-                                    <th className="cell-fit" style={{"width": "60%"}}>
-                                        {"Addresses " + (row_start + 1) + " to " + row_stop}
-                                    </th>
-                                    <th className="cell-fit" style={{"textAlign": "center", "width": "20%"}}>
-                                        {"Reputation"}
-                                    </th>
-                                    <th className="cell-fit" style={{"textAlign": "center", "width": "20%"}}>
-                                        {"Eligibility"}
-                                    </th>
-                                </tr>
-                            </thead>
-                            <tbody>
-                                {this.state.reputation_rows.slice(row_start, row_stop)}
-                            </tbody>
-                        </Table>
-                    </Card.Text>
+            <Card className="shadow bg-white mb-4 rounded">
```

Comment:
Can we change from two columns layout to single column before Eligibility disappears?
![image](https://github.com/user-attachments/assets/f482be3f-492b-4c1b-88b3-55c056620174)

Reply:
Somewhat irrelevant to still work on this as the page will disappear with wit/2.